tuner/tuner/common_test.py

```diff
@@ -109,7 +109,8 @@ def test_get_compatible_mfma_intrinsics(tuner_ctx: common.TunerContext) -> None:
             common.ShapedType([1280, 1280], tuner_ctx.type.f16),
             common.ShapedType([2048, 1280], tuner_ctx.type.f32),
             common.DispatchKind.mmt,
-        )
+        ),
+        [],
```

Comment:
We should make sure the new logic is covered by tests. Please add/modify existing tests to pass in the list of available intrinsics.